### DISCUSSION TITLE
About non-OSS Repository

### DISCUSSION THREAD

**Discussion by MizuiMiduki** [community/community#36933] [Other Feature Feedback, Questions, & Ideas]
I'm not good at English, so I use translation software.

Wikipedia states that GitHub "provides free accounts for open source projects" and "hosts source code", but only compiled versions are published and the source code cannot be viewed. There are things that are not.
Would such a repository violate GitHub's Terms of Service?

**Comment by ossanna16** [comment]
Hi there @MizuiMiduki and welcome to our community 😄. Thanks so much for asking a great question!
